scripts/docker/Dockerfile.sycl

```diff
@@ -1,16 +1,18 @@
-ARG BASE=intel/oneapi-basekit:devel-ubuntu18.04
+ARG BASE=nvidia/cuda:10.2-devel
 FROM $BASE
 
 RUN apt-get update && apt-get install -y \
+        bc \
         wget \
         ccache \
+        ninja-build \
+        python3 \
+        git \
         && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
-ENV PATH=/opt/intel/oneapi/compiler/latest/linux/bin/:$PATH
-
-ARG CMAKE_VERSION=3.10.3
+ARG CMAKE_VERSION=3.19.0
```

Comment:
Is version 3.19 required or did you grab the latest release available?

Reply:
I grabbed the latest release since 3.10.3 wasn't sufficient. Similar to what you did for `OpenMPTarget`.